doc/services/device_mgmt/mcumgr.rst

```diff
@@ -43,9 +43,21 @@ Command-line Tool
 MCUmgr provides a command-line tool, :file:`mcumgr`, for managing remote devices.
 The tool is written in the Go programming language.
 
-To install the tool::
+To install the tool:
 
-  go get github.com/apache/mynewt-mcumgr-cli/mcumgr
+.. tabs::
+
+   .. group-tab:: go version <= 1.17
```

Comment:
Shouldn't this be `go version < 1.18.xx` and the other one `go version >= 1.18.xx`?
Even when you left out the minor part (`xx`) it seems more correct than using 1.17 as reference point, since actual reference point is 1.18.